index.html

```diff
@@ -0,0 +1,332 @@
+                                </div>
+                                <div class="codingLangBackground">
+                                    Ruby on Rails
+                                </div>
+                                <div class="codingLangBackground">
+                                    JavaScript
+                                </div>
+                            </div>
+                            <div class="seeThisProject">
+                                <img class="stpButton" src="/img/stpButton.png" alt="geometryDivisor">
+                            </div>
+
+                        </div>
+                    </div>
+                </div>
+
+
+                <div class="projectBox">
+                    <div class="boxContainer">
+                        <div class="picture">
+                            <img class="picturesPreview" src="/img/pic3.png" alt="picture1">
+                        </div>
+                        <div class="textBox">
+                            <div class="boxTitle">
+                                Project name goes here
+                            </div>
+                            <div class="codingLangButtons">
+                                <div class="codingLangBackground">
+                                    HTML/CSS
+                                </div>
+                                <div class="codingLangBackground">
+                                    Ruby on Rails
+                                </div>
+                                <div class="codingLangBackground">
+                                    JavaScript
+                                </div>
+                            </div>
+                            <div class="seeThisProject">
+                                <img class="stpButton" src="/img/stpButton.png" alt="geometryDivisor">
+                            </div>
+
+                        </div>
+                    </div>
+                </div>
+
+
+                <div class="projectBox">
+                    <div class="boxContainer">
+                        <div class="picture">
+                            <img class="picturesPreview" src="/img/pic4.png" alt="picture1">
+                        </div>
+                        <div class="textBox">
+                            <div class="boxTitle">
+                                Project name goes here
+                            </div>
+                            <div class="codingLangButtons">
+                                <div class="codingLangBackground">
+                                    HTML/CSS
+                                </div>
+                                <div class="codingLangBackground">
+                                    Ruby on Rails
+                                </div>
+                                <div class="codingLangBackground">
+                                    JavaScript
+                                </div>
```

Comment:
- [ ] kindly consider adding The technology in each category in a list `ul` and each of them a `li` this will help you to fulfill this requirement 👇 
> Please use ul and li for categories in the "Work" section. Html, CSS, and Javascript should be list items.